### DISCUSSION TITLE
How to share queries from https://shell.duckdb.org/ with ".timer on"

### DISCUSSION THREAD

**Discussion by gregorywaynepower** [duckdb/duckdb#15402] [Q&A] [answered]
Hey Folks,

Trying to showcase DuckDB's performance and I really enjoy the .timer dot function, anyone got a PRAGMA that'll work in the shell.

https://shell.duckdb.org/#queries=v0,.timer-on Returns a syntax error.

**Comment by carlopi** [comment, marked as answer]
I am not sure if there is a workaround, but that can be added, maybe moving as a duckdb/duckdb-wasm issue? Thanks!

**Reply by gregorywaynepower** [reply]
Looks like someone is one step ahead of me.

https://github.com/duckdb/duckdb-wasm/issues/1917

**Reply by carlopi** [reply]
That @szarnyasg is not to be messed with. Thanks for raising, again, I will have a look in the next weeks
